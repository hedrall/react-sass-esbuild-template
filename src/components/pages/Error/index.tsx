import React from "react";

export type ErrorProps = {};

export const Error: React.FC<ErrorProps> = (props) => {
  return <div className="Error">Error page</div>;
};
