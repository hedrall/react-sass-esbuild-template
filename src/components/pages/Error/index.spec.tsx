import React from "react";
import { render, screen } from "@testing-library/react/pure";
import { Error, ErrorProps } from "@frontend/components/pages/Error/index";

describe("Error", () => {
  let container: HTMLElement;

  const props: ErrorProps = {};

  beforeAll(() => {
    const tree = render(<Error {...props} />);
    container = tree.container;
  });

  test("snapshot", () => {
    expect(container).toMatchSnapshot();
  });
});
