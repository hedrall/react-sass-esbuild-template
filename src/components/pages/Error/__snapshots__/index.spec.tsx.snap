// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error snapshot 1`] = `
<div>
  <div
    class="Error"
  >
    Error page
  </div>
</div>
`;
